test/test_structure_element2.py

```diff
@@ -45,17 +46,17 @@ def test_set_length_key(self):
         se = self.setup_structure_element(p, 2, 1, 5, 0)
 
         self.assertEqual(se.get_length(), 5)  
-        se.set_length_key(7)
-        self.assertEqual(se.get_length(), 7)       
+#        se.set_length_key(7)
+#        self.assertEqual(se.get_length(), 7)       
```

Comment:
Disabling tests is usually a very very bad idea. Often a test is a result of hours spent debugging, and nobody wants to do that again. If you can't make the test pass, decorate it with `@unittest.expectedFailure`.